metacat-main/build.gradle

```diff
@@ -29,9 +29,12 @@ dependencies {
     compile(project(":metacat-common"))
     compile(project(":metacat-common-server"))
     compile(project(":metacat-thrift"))
-    compile("org.elasticsearch:elasticsearch")
+    compile("org.elasticsearch.client:transport")
+
     compile("com.amazonaws:aws-java-sdk-sns")
     compile("joda-time:joda-time")
+    compile("org.apache.logging.log4j:log4j-api")
```

Comment:
Is there a reason we're using log4j like this here instead of using slf4j/logback like everywhere else?

Reply:
Then why do we have to explicitly depend on it. It does not get it transitively

Reply:
It may change based on the spring boot version.
want to set it to a specific version, which the cluster is.